app/controllers/hostgroups_controller.rb

```diff
@@ -103,6 +112,10 @@ def process_hostgroup
     render :partial => "form"
   end
 
+  def csv_columns
+    [:name, :operatingsystem, :environment, :hosts_count, :children_hosts_count]
```

Comment:
We don't show os or environment in the ui, so we shouldn't export them in the csv either.